addon/components/nrg-new-features/component.js

```diff
@@ -1,26 +1,35 @@
 import Component from '@ember/component';
 import { inject as service } from '@ember/service';
-import { alias, bool, reads } from '@ember/object/computed';
-import { task } from 'ember-concurrency';
+import { computed } from '@ember/object';
+import { alias } from '@ember/object/computed';
+import { htmlSafe } from '@ember/string';
 import layout from './template';
 
 export default Component.extend({
   layout,
 
   whatsNewService: service('whats-new'),
 
-  hasContent: bool('content'),
-  isOpen: reads('hasContent'),
+  isOpen: false,
 
   headerText: alias('whatsNewService.headerText'),
 
   primaryButtonText: alias('whatsNewService.primaryButtonText'),
   secondaryButtonText: alias('whatsNewService.secondaryButtonText'),
+  htmlContent: computed('content', function() {
+    return htmlSafe(this.content);
+  }),
 
-  checkContent: task(function*() {
-    let content = yield this.get('whatsNewService').getContent();
+  didInsertElement() {
+    this._super(...arguments);
+    this.checkContent();
+  },
+
+  async checkContent() {
+    let content = await this.get('whatsNewService').getContent();
     this.set('content', content);
-  }).on('init'),
+    this.set('isOpen', !!content);
```

Comment:
Why revert using the `bool` and `reads` paradigm? I'm not a big fan of this `!!` .